Packs/Darktrace/Integrations/Darktrace/Darktrace_test.py

```diff
@@ -0,0 +1,316 @@
+import json
+import io
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+def test_get_breach(requests_mock):
```

Comment:
unit tests should contain given-when-then comment to describe the unit test scenario. Given <some input>, When <some state> Then <ensure some behviour>

Reply:
Added comments for each unit test